src/bridge/cockpitblocksamples.c

```diff
@@ -88,7 +88,7 @@ cockpit_block_samples (CockpitSamples *samples)
        */
 
       num_parsed = sscanf (line,
-                           "%d %d %64s"
+                           "%d %d %127s"
```

Comment:
Good enough for 133, but general it would be more robust to use `%n$ms` and a `gchar **dev_name;` so that `sscanf()` allocates the buffer by itself. Then we don't need to assume any maximum length any more.